### DISCUSSION TITLE
editor - adjusting slider anchors while playing destroys ears

### DISCUSSION THREAD

**Discussion by Espei-on-osu** [ppy/osu#19714] [Q&A] [answered]
when I created a long slider, I played the song to see how it will move, and i started adjusting the slider when the slidertick sound absolutely destroyed my ears. (I am editing on macOS Big Sur, not sure if it is a problem on other platforms)

It scared me and I think that should be changed.

[database.log](https://github.com/ppy/osu/files/9312917/database.log)
[network.log](https://github.com/ppy/osu/files/9312919/network.log)
[performance.log](https://github.com/ppy/osu/files/9312920/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9312921/runtime.log)

**Comment by peppy** [comment, marked as answer]
Thanks for the report and sorry for any ear damage 😅 .

This should be fixed via #19800.
